src/templates/static/ib_programme/ib_signup.jsx

```diff
@@ -49,6 +49,7 @@ const Signup = () => (
             <h2 className='center-text'>{it.L('Commission structure')}</h2>
             <div className='gr-padding-20 center-text'>
                 <Table
+                    className='ib-signup-page__commission-table'
```

Comment:
I don't think we have this style of `__`, why not just use `commission-table` and then in sass file call it by `.ib-signup-page { .commission-table ...`

Reply:
oh did you already change the code, I'm going through each commit. but there'a  lot of `--` and `__` if we want to use it we can, we just need to add documentation for it so we consistently use it everywhere

Reply:
also which documentation you were talking about? :) And probably we should consult with Mohammad first

Reply:
documentation could be added to a readme file in the sass folder.